develop-docs/application-architecture/dynamic-sampling/fidelity-and-biases.mdx

```diff
@@ -13,19 +13,20 @@ A sample rate is a number in the interval `[0.0, 1.0]` that will determine the l
 
 ## The Concept of Fidelity
 
-At the core of Dynamic Sampling there is the concept of **fidelity**, which translates to an overall **target sample rate** that should be applied across all transactions of an organization.
+At the core of Dynamic Sampling there is the concept of **fidelity**, which translates to an overall **target sample rate** that should be applied across all events of an organization.
```

Comment:
Today, it applies to some of our tracing data but not all. DS applies to spans and transaction events, but not to error  and message events from error monitoring. What further complicates this is that technically errors are a part of the trace, and as such the trace definition you deliver below is absolutely correct.

Note that also in many parts of the code base "Event" refers to a base type for the different kind of data collected through error monitoring (error, security, message), whereas spans have a different root type. Transaction events used to share that same base type for historical reasons, but over time were considered something else until we started considering their deprecation.